people.js

```diff
@@ -14,7 +14,7 @@ const getPeople = async function getPeople() {
 function checkIsValidDomain(emailDomain) {
   if (emailDomain === undefined) throw "Error: Input is undefined";
   if (typeof(emailDomain) !== "string") throw "Error: Input is not in proper format";
-  if (emailDomain.trim().length = 0) throw "Error: Empty spaces not allowed";
+  if (emailDomain.trim().length == 0) throw "Error: Empty spaces not allowed";
```

Comment:
Thanks for checking the error type handling. The change fixed the error on eslint and will make my code run better!